templates/zerver/help/private-messages.md

```diff
@@ -39,3 +39,8 @@ There are several ways to access an existing PM or group PM.
   able to select PMs or group PMs with that user.
 * Open the compose box, and enter in a (list of) users. Type `Ctrl + .` (or
   `Cmd + .` on a mac) to open that conversation.
+
+Note that we show (<span class="fa fa-group"></span>) icon for [Group PMs](/help/private-messages)
+and colored dot (like <span class="indicator green solid"></span>) for 1:1 PMs as per the
+[status and availabily](/help/status-and-availability) of the user to differentiate between
```

Comment:
![image](https://user-images.githubusercontent.com/59444243/117319988-7163bd00-aea9-11eb-9395-68647ece2f02.png)

Reply:
I think this documentation is unnecessary; anyone looking at the left sidebar with a group PM in there should be able to understand it.